src/modules/organisations/organisations.controller.ts

```diff
@@ -84,4 +88,47 @@ export class OrganisationsController {
     const { sub } = req.user;
     return this.organisationsService.getOrganisationMembers(org_id, page, page_size, sub);
   }
+
+  @ApiOperation({ summary: 'Get members of an Organisation' })
+  @ApiResponse({
+    status: 200,
+    description: 'Assign roles to members of an organisation',
+    schema: {
+      properties: {
+        status: { type: 'string' },
+        message: { type: 'string' },
+        data: {
+          type: 'object',
+          properties: {
+            user: { type: 'string' },
+            org: { type: 'string' },
+            role: { type: 'string' },
+          },
+        },
+      },
+    },
+  })
+  @ApiResponse({
+    status: 404,
+    description: 'Organisation not found',
+  })
+  @ApiResponse({
+    status: 403,
+    description: 'User not a member of the organisation',
+  })
+  @Put(':orgId/members/:memberId/role')
+  async updateMemberRole(
+    @Param('memberId') memberId: string,
+    @Param('orgId') orgId: string,
+    @Body() updateMemberRoleDto: UpdateMemberRoleDto
+  ) {
+    try {
```

Comment:
Remove try...catch.

We catch all `internalServerError` exception and advise not to `throw`